src/test/java/com/ibm/cloudant/kafka/connect/CloudantSourceAndSinkTest.java

```diff
@@ -88,13 +92,17 @@ public void testReplicateAll() throws Exception {
             // - no offset
             for (SourceRecord record : records) {
 
+                // source task returns strings but sink task expects structs or maps
+                // in a real kafka instance this would be fixed by using appropriate converters
+                Map recordValue = gson.fromJson((String)record.value(), Map.class);
```

Comment:
This is a bit awkward but is needed until we do the PR to support JsonConverter in the source connector - at which point this line can be removed since the source will return us a `Map`

Reply:
@ricellis your idea of enabling schemas here (to return a struct/map instead of a string) didn't work because it caused various limits to be exceeded (memory, http request size), because the test payloads are complex resulting in a huge inline schema per doc.